src/CommonITILObject.php

```diff
@@ -10759,14 +10759,14 @@ public static function cronCreateInquest($task)
     }
 
     /**
-     * Returns the {@link CommonITILSatisfaction} class for the current itemtype
-     * @return class-string<CommonITILSatisfaction>|null
+     * Returns the {@link CommonITILSatisfaction} class instance for the current itemtype
+     * @return CommonITILSatisfaction|null
      */
-    public static function getSatisfactionClass(): ?string
+    public static function getSatisfactionClassInstance(): ?CommonITILSatisfaction
```

Comment:
If this kind of method can return `null`, that implies the result to be tested at usage; which seems not a good idea. I'm almost sure we do not expect a null most of the time.
Throwing a specific exception seems better, and this one can be catched when this is really not an issue.

Reply:
Satisfaction is used on tickets and changes but not on problems. If we throw an exception here, we will have to add a `hasSatisfactionClass()` method or something similar.